source/admin_manual/list_of_events.rst

```diff
@@ -951,8 +953,8 @@ requests (e.g. doveadm HTTP API).
    :added: v2.3.18
    :inherit: http_server
 
-   :field bytes_in: Amount of request data read, in bytes.
-   :field bytes_out: Amount of response data written, in bytes.
+   :field net_in_bytes: Amount of request data read, in bytes.
+   :field net_out_bytes: Amount of response data written, in bytes.
```

Comment:
maybe all these should have `@added` or `@changed`?

Reply:
I'll add @changed...